BML_TUX_Project/Packages/bmlTUX/Scripts/ExperimentParts/ExperimentRunner.cs

```diff
@@ -95,7 +95,7 @@ public abstract class ExperimentRunner : MonoBehaviour {
             }
             DesignFile.GetValidate();
 
-            Session = Session.LoadSessionData();
+            Session = Session.LoadSessionData(DesignFile.GetFileLocationSettings);
```

Comment:
I don't think we need GetFileLocationSettings anymore, we can just reference the FileLocationSettings scriptable object directly (see below)

Reply:
Not quite sure what you mean, I believe this is still needed?